### ISSUE TITLE
Add VMBRF8S

### ISSUE THREAD

**Issue by MDAR** [cedricboon/openhab2-addons#18]
Hi

There is a new module on the block, called a VMBRF8S

As far as support goes, it's nothing fancy, just an 8 input module, like the VMB8PBU
As soon as I can get hold of a link to the Protocol document, I'll add it here.

(I've got a module on my home network if you want to take a peek at it, at Address EF)

Many thanks.

**Comment by MDAR** [comment]
Hi

Just because I wanted to try...

I have added this module to my setup using a VMB8PBU profile.

The button events are detected in events.log and the alarm (1) state, type and times are all correct.
(I haven't tried Alarm (2) yet, but I'll assume they are fine)

Obviously there isn't any LED feedback on this module.

**Comment by MDAR** [comment]
Hello 

The Velbus team have added the protocol document for this module to their GitHub page

https://github.com/velbus/moduleprotocol/blob/master/protocol_vmbrfr8s.pdf

**Comment by cedricboon** [comment]
It looks almost identical to the vmb8pbu.
Can you try [this](https://github.com/cedricboon/openhab2-addons/releases/tag/2.5.2.202002151409) version?

**Comment by MDAR** [comment]
Hi

It's certainly found the VMBRF8S :smile: 


My first observations are that the VMBRF8S doesn't have any Feedback LEDs.

And you've removed the word Wakeup from the alarm 1 Wake  minute..

![Screenshot_20200216-095733_Chrome](https://user-images.githubusercontent.com/10905891/74602670-132f9c00-50a3-11ea-8ae0-859ca3ac6dd3.jpg)


I'll be busy for the rest of today, so I'll look at it in more detail tomorrow when I'm in the office.

Have a great weekend / Sunday.

**Comment by MDAR** [comment]
Hi

I've just tried to add the binding to a system and it can't detect his Edge Lit OLED panel.

I've rolled him back to version 202001110742 and that has the same problem.

If I add the VMBELDO manually, it says that the Velbus Bridge is OFFLINE, which is really odd because everything else is working perfectly.

I've restarted the binding

restarted openhab2

the only strange thing is that the are a load of Things that are taking a long time to delete.

I'll finish the house-keeping and see if the situation improves

**Comment by MDAR** [comment]
Hi

This may be unconnected, but I'm getting a lot of disconnections between openHAB2 and Velbus.

I'm trying to work out where the issue is, as I've also upgraded to Oh2.5.2-2 (days before you released this version.

I'll roll back each part and see what happens.

**Comment by MDAR** [comment]
I think something has gone very wrong with my openHAB2 configuration.

So I'm going to start again from scratch.

The VMBRF8S seems to be working well, with the minor details mentioned above.


**UPDATE**

I think I've found (remembered) the issue.

My house openHAB2 is running on an Odroid XU4, which has 4 Little cores and 4 Big cores.
When I first set it up, openHAB2 crashed randomly, just like it is / was now.

My fix was to Taskset the openHAB2 Java to the Big cores


Obvious. I'd completely forgotten about the edit I made to the openhab2.service file when I last upgraded openHAB2.
